docs/guides/joint_fabric_guide.md

```diff
@@ -250,4 +253,16 @@ jf-admin-app has been installed:
 A `Subjects` field equal to `18446744065119551489` (`FFFFFFFDFFFF0001` in hex)
 should be found.
 
+-   Open Joint Commissioning Window on JF Admin App of Ecosystem B
```

Comment:
Please move this new text inside the Run Joint Fabric Demo section.

Reply:
Addressed in latest commit.